src/styles/body.scss

```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: #a78989;
+}
```

Comment:
you could combine this and styles below where you apply box-sizing into one file